src/test/java/com/github/beatngu13/visualtesting/util/ElementCount.java

```diff
@@ -0,0 +1,16 @@
+package com.github.beatngu13.visualtesting.util;
+
+import org.openqa.selenium.WebDriver;
+
+import de.retest.web.selenium.By;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ElementCount {
+
+	public ElementCount() {}
+
+	public void extract( WebDriver driver ) {
+		log.info( "Html file contains {} elements.", driver.findElements( By.xpath( "//*" ) ).size() );
```

Comment:
```suggestion
		log.info( "Current page contains {} WebElements.", driver.findElements( By.xpath( "//*" ) ).size() );
```

I would also extract the actual counting:

```java
private int getWebElementCount( WebDriver driver ) {
    List<WebElement> allElements = driver.findElements( By.xpath( "//*" ) );
    return allElements.size();
}